### PR TITLE
[new release] tracy and tracy-client (0.1)

### PR DESCRIPTION
**Pull request by c-cube**
Virtual package for the Tracy profiler https://github.com/wolfpld/tracy

- Project page: <a href="https://github.com/imandra-ai/ocaml-tracy">https://github.com/imandra-ai/ocaml-tracy</a>

##### CHANGES:

initial release

### CODE REVIEW

**Comment by c-cube** [comment]
Seems like the lower bounds is an issue with using a version of dune that's too old?

**Comment by avsm** [comment]
Then increase the lower bound of dune until you have one that works?

**Comment by c-cube** [comment]
I was hoping opam-ci would do that :). I'll try and fix that, but it's a bit of a pain since changing dune's version means recompiling the whole switch _every time_ :worried:

**Comment by c-cube** [comment]
Well well well, it was all transient failures, apparently. I couldn't reproduce the failure locally, and a rebuild fixed them. The remaining ones are on alpine which might not be able to even _run_ Tracy as is? Compilation fails in the C++ bindings.

File: packages/tracy-client/tracy-client.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+license: "BSD-3-Clause"
+homepage: "https://github.com/imandra-ai/ocaml-tracy"
+bug-reports: "https://github.com/imandra-ai/ocaml-tracy/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "ocaml" {>= "4.08"}
+  "tracy" {= version}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/tracy-client/tracy-client.0.1/opam:15]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/tracy/tracy.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+authors: ["Simon Cruanes"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/imandra-ai/ocaml-tracy"
+bug-reports: "https://github.com/imandra-ai/ocaml-tracy/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "ocaml" {>= "4.08"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/tracy/tracy.0.1/opam:14]:
```suggestion
  ["dune" "subst"] {dev}
```

**Pull request closed without merging**